app/Http/Controllers/Entity/PostController.php

```diff
@@ -25,12 +25,20 @@ public function create(Campaign $campaign, Entity $entity, Post $post)
     {
         $this->authorize('post', [$entity->child, 'add']);
         $parentRoute = $entity->pluralType();
+        $templates = Post::template()->pluck('name', 'id')->all();
+
+        $template = request()->input('template');
+        if (!empty($template)) {
+            $template = Post::template()->where('id', $template)->first();
```

Comment:
As a non-admin, can I can call the url manually with a post id and magically get everything? 🫣